doc/man-template.roff

```diff
@@ -1,45 +1,143 @@
+
+xh uses HTTPie's request-item syntax to set headers, request body,
+query string, etc. For more details, refer to REQUEST_ITEM in the
+OPTIONS section.
+
+If stdin is detected as not tty, xh will attempt to read the request body
+from stdin (you might want to opt-out of this behaviour in non-interactive
+sessions via \-\-ignore\-stdin).
+
+.RS
+.EX
+$ cat report.txt | xh post example.com
+.EE
+.RE
+
+By default, rustls is used instead of the system's TLS library. However,
+one can opt-out by passing the \-\-native\-tls option (requires compiling
+xh with the native\-tls feature).
+
+xh can be configured via ~/.config/xh/config.json. The only configurable
+option is "default_options" which is a list of default shell arguments
+that gets passed to xh.
+
+.RS
+.EX
+{
+  "default_options": ["--native-tls", "--style=solarized"]
+}
+.EE
+.RE
+
+.SH OPTIONS
+Each of the following options can be reset with a --no-OPTION argument.
+
+{{options}}
+
+.SH EXIT STATUS
+.TP
+.B 0
+Successful program execution.
+.TP
+.B 1
+Usage, syntax or network error.
+.TP
+.B 2
+Request timeout.
+.TP
+.B 3
+Unexpected HTTP 3xx Redirection.
+.TP
+.B 4
+HTTP 4xx Client Error.
+.TP
+.B 5
+HTTP 5xx Server Error.
+.TP
+.B 6
+Too many redirects.
+
+.SH ENVIRONMENT
+.TP
+.B XH_CONFIG_DIR
+Specifies where to look for config.json and named session data.
+The default is $XDG_CONFIG_HOME/xh.
+.TP
+.B XH_HTTPIE_COMPAT_MODE
+Enables the HTTPie Compatibility Mode. The only current difference is that
+\-\-check-status is not enabled by default. An alternative to setting this
+environment variable is to rename the binary to either http or https.
+.TP
+.BR REQUESTS_CA_BUNDLE ", " CURL_CA_BUNDLE
+Sets a custom CA bundle path.
+.TP
+.B HTTPS_PROXY
+Sets the proxy server to use for HTTPS.
+.TP
+.B http_proxy
+Sets the proxy server to use for HTTP.
+.TP
+.B NETRC
+Location of the .netrc file.
+.TP
+.B NO_COLOR
+Disables output coloring. See <https://no-color.org>
+
+.SH FILES
+.TP
+.I ~/.config/xh/config.json
+xh configuration file.
+.TP
+.IR ~/.netrc ", " ~/_netrc
+Auto-login information file.
+.TP
+.I ~/.config/xh/sessions
+Session data directory grouped by domain and port number.
+
+.SH EXAMPLES
+.TP 4
+xh \fIhttpbin.org/json\fR
+Send a GET request.
+.TP
+xh \fIhttpbin.org/post name=ahmed \fIage:=24\fR
+Send a POST request with body {"name": "ahmed", "age": 24}.
+.TP
+xh get \fIhttpbin.org/json id==5 sort==true\fR
+Send a GET request with querystring id=5&sort=true.
+.TP
+xh get \fIhttpbin.org/json x-api-key:12345\fR
+Send a GET request and include a header named x-api-key with value 12345.
+.TP
+xh put \fIhttpbin.org/put id:=49 age:=25\fR | less
+Send a PUT request and pipe the result to less.
+.TP
+xh -d \fIhttpbin.org/json\fR -o \fIres.json\fR
+Download and save to res.json.
+.TP
+xh \fIhttpbin.org/get user-agent:foobar\fR
+Make a request with a custom user agent.
+
+.SH REPORTING BUGS
+xh's Github issues <https://github.com/ducaale/xh/issues>
+
+.SH SEE ALSO
+curl(1), http(1)
```

Comment:
I suggest that the command name in bold. This is inspired by [`curl(1)`](https://curl.se/docs/manpage.html) and [`man-pages(7)`](https://man7.org/linux/man-pages/man7/man-pages.7.html).